mlflow/java/scoring/pom.xml

```diff
@@ -107,6 +115,17 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
It looks like `maven-source-plugin` and `maven-shade-plugin` are both included here for building a scoring jar. Does this current configuration of the maven source plugin create an uber jar containing all of the scoring package? If so, it seems like we can remove the shade plugin above.

Reply:
`maven-source-plugin` just builds a source-jar so that people can download the sources (code) through maven. It does not include the binary artifacts.